ui-tests/src/test/java/io/syndesis/qe/pages/apps/todo/Todo.java

```diff
@@ -0,0 +1,13 @@
+package io.syndesis.qe.pages.apps.todo;
+
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.$$;
+
+import org.openqa.selenium.By;
+
+public class Todo {
+
+    public int getListItemsCount() {
```

Comment:
Please add comment for this method that it is not count of items in todo app, but count of 'list-group-item' elements on the page. So even with just one element inside of todo app method will return 4.

Reply:
thanks, I didn't notice that. I've changed xpath for the method to do what it says.